modules/oe_paragraphs_illustrations_lists/README.md

```diff
@@ -0,0 +1,17 @@
+OpenEuropa Paragraphs Illustration Lists
+============================
+
+This module provides Illustration list paragraphs that depend on the [OpenEuropa Media](https://github.com/openeuropa/oe_media)
+component and the [Description List Field](https://www.drupal.org/project/description_list_field) module.
```

Comment:
As I understand we can drop dependency from `description_list_field` as we don't use this module anymore.